package.nls.json

```diff
@@ -23,6 +23,9 @@
 	"command.update": "Update to...",
 	"command.branch": "Create Branch...",
 	"command.pull": "Pull",
+	"command.purge": "Purge",
+	"command.purgeFiles": "Purge All Untracked Files",
```

Comment:
What is the difference between "Purge" and "Purge All Untracked Files"?

Reply:
Purge is for the individual file, and Purge all untracked is for the whole untracked files source group.
This allow the user to have a fine grain control on the Untracked Source Group view.